15_merge_inplace.md

```diff
@@ -39,7 +39,7 @@ It was very sad.
 But, we often reinvent things.
 
 Meanwhile Knuth heard about it from his friend
-[Ian Pratt][pratt] who told him about it.
+[Vaughan Pratt][pratt] who told him about it.
```

Comment:
I believe you are correct, but I am curious if you could just hear the audio better, or have a reason for knowing this.

Reply:
Yes it's an easy mistake since the names are so similar, but I remembered that Alex jokes in another video (I think it was in the Programming Conversations course) that when Knuth doesn't know who invented an algorithm, he always gives credit to Vaughn Pratt, so I was curious about who this guy was (I actually first though he said Von Pratt and that it was a made-up person :) ) and then realized he is must be same guy from the famous Knuth-Morris-Pratt algorithm...

What Alex refers to here is actually in the answer to exercise 5.5.3 in The Art of Computer Programming vol. 3 (see pages 390 and 701).

Reply:
Fantastic, I will give that exercise a read and add a footnote with this information.